### DISCUSSION TITLE
Reset a game instead of going back to the menu

### DISCUSSION THREAD

**Discussion by Kei01g** [Polprzewodnikowy/N64FlashcartMenu#129] [Q&A]
When I hit the reset button on the console I want to reset a game instead of going back to N64FlashcartMenu
I'm using SummerCart64 and I read the flashcart's documentation but couldn't find how to change this behaviour

**Comment by Polprzewodnikowy** [comment]
It's not implemented yet https://github.com/Polprzewodnikowy/N64FlashcartMenu/issues/75

**Reply by Kei01g** [reply]
Thank you for the fast reply, also I'm sorry I should've read the issues

**Comment by networkfusion** [comment]
This has now been implemented in the latest menu.
